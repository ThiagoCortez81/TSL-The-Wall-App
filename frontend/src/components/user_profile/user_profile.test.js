import React from 'react';
import { shallow } from 'enzyme';
import UserProfile from './user_profile';
describe('UserProfile', () => {
    it('should render correctly in "debug" mode', () => {
        const component = shallow(<UserProfile debug/>);

        expect(component).toMatchSnapshot();
    });
});