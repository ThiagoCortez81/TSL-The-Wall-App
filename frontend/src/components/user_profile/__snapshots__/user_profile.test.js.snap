// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserProfile should render correctly in "debug" mode 1`] = `
<div
  className="card wall-post"
>
  <Col
    bsClass="col"
    className="p-0 bg-white"
    componentClass="div"
    md={12}
  >
    <Col
      bsClass="col"
      className="p-0 bg-white text-center"
      componentClass="div"
      md={12}
    >
      <h4
        className="fw-6"
      >
        Please, log in
      </h4>
      <p>
        <NavLink
          activeClassName="active"
          aria-current="page"
          to="/auth"
        >
          Log in
        </NavLink>
      </p>
    </Col>
  </Col>
</div>
`;
